frontend/amundsen_application/static/js/interfaces/Feature.ts

```diff
@@ -43,3 +39,9 @@ export interface FeatureSummary {
 export interface FeatureStats {
   name: string;
 }
+
+export interface FeatureWatermark {
+  key: string;
+  watermark_type: string;
+  time: string;
+}
```

Comment:
Is the different watermark format inherited from some upstream source or are have we created this format ourselves? I'd prefer to reuse the existing watermark, or to change both of them to the new format. WDYT?

Reply:
We could map the new feature watermark model dmitriy made to the existing one in FE I believe, but there is no column name